blog/app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = t(".intro")
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
```

Comment:
dùng toán tử 3 ngôi, so sánh giúp a: nil?, empty?, blank?, present?
ở đây nên dùng cái nào

Reply:
nil?
- Có thể sử dụng trên bất kỳ đối tượng nào 
- Sẽ chỉ trả về true khi thực sự là nil.
empty?
-Khác với nil?, empty? chỉ sử dụng trên các đối tượng dạng tập hợp (collection) như Hash, array.
- Phương thức sẽ trả về true khi tập hợp không có phần tử nào.
blank?
- nil , false, một khoản trắng rõ ràng là bank còn true thì không.
- Đối với Array và Hash thì là blank? khi chúng empty.
-Khác với String#empty? chỉ kiểm tra trên độ dài của chuỗi thì String#blank? còn kiểm tra thêm cả khoảng trắng.
present?
- present? là phủ định của blank? và có thể sử dụng ở bất cứ đâu
=> Em thấy trong trường hợp này nên sử dụng blank?